### ISSUE TITLE
Charles Proxy issues

### ISSUE THREAD

**Issue by Ahdrumond** [mdesilva/AmazonFlexUnlimited#134]
Is anyone also having issues with Amazonflex App and Charles Proxy?

**Comment by vtdeleond** [comment]
Yes. I'm having some issue. Seems like Amazon Flex has taken some action adding some Amazon Certification at their end.

**Comment by Mankaran4466** [comment]
> Yes. I'm having some issue. Seems like Amazon Flex has taken some action adding some Amazon Certification at their end.

I really appreciate the effort of Gigwebtech in reactivating my Amazonflex driver and spark driver account. They are indeed lifesavers, saving my job by getting my account reactivatedhttps://t.me/sparkdriverss/
